src/config.rs

```diff
@@ -378,6 +378,111 @@ impl de::Deserialize for ActionWrapper {
     }
 }
 
+struct CommandWrapper {
+    program: String,
+    args: Vec<String>,
```

Comment:
This should probably be optional, so add a `#[serde(default)]` (the default is an empty vec).